rnd/autogpt_server/.env.example

```diff
@@ -57,5 +57,4 @@ MEDIUM_AUTHOR_ID=
 LOG_LEVEL=INFO
 ENABLE_CLOUD_LOGGING=false
 ENABLE_FILE_LOGGING=true
-ENABLE_CONSOLE_LOGGING=true
-ENABLE_JSON_LOGGING=false
+LOG_DIR=./logs
```

Comment:
What do you think this resolves to when `LoggingConfig` parses `./logs` into a `Path`?

Reply:
Relative to the directory python is called form I think.

I added it into examples as this is examples But maybe I should comment it out so the default behaviour is to use the hardcoded value

Reply:
You're probably right. In that case having `./logs` as the default would be fine.